code/modules/hydroponics/hydro_tray_process.dm

```diff
@@ -227,12 +227,17 @@
 
 /obj/machinery/portable_atmospherics/hydroponics/proc/check_gasses(var/datum/gas_mixture/environment)
 	// Handle gas consumption.
+	// If it has the absorbing trait, takes no damage from lack of gas
 	if(seed.consume_gasses && seed.consume_gasses.len && environment)
 		missing_gas = 0
 		for(var/gas in seed.consume_gasses)
 			if(environment[gas] < seed.consume_gasses[gas])
-				missing_gas++
+				if(!seed.gas_absorb)
+					missing_gas++
 				continue
+			if (seed.gas_absorb)
+				seed = seed.diverge(1)
+				seed.potency += 0.2
```

Comment:
lack of a cap or softcap here is possibly bad given other potency methods do incorporate those
although it would be a fair amount of work to farm cabbages in this way, not all gas consuming plants are so dangerous - for example, vox chicken of the stars consumes nitrogen and it's going to be naturally self-buffing at all times

Reply:
Funnily enough, this would be the only legitimate way to get 200 potency instead of the scuffed 198 we have currently, cause my ghetto fix to just bump the potency to 200 was rejected, and I don't understand DM wizardry enough to tweak the formula correctly.